rubrik_cdm/data_management.py

```diff
@@ -885,3 +886,81 @@ def get_sla_objects(self, sla, object_type, timeout=15):
                         sla, object_type))
 
             return vm_name_id
+
+    def time_in_range(self, start, end, x):
```

Comment:
`time_in_range` updates:

* function should be internal (i.e named `_time_in_range`)
* let's use a more descriptive variable name than `x`
* need full docstring similar to the rest of the functions